README.md

```diff
@@ -10,6 +10,17 @@ Debezium is a distributed platform that turns your existing databases into event
 
 Debezium is open source under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
 
+# Purpose of the Debezium UI
+The Debezium UI provides standalone web application, which connects to Kafka Connect via its REST API. See the [Debezium UI docs](https://debezium.io/documentation/reference/operations/debezium-ui.html) for more information
+
+# Debezium environment vars
+
+| ENV vars        | application.properties vars  | description |
+| :-------------- |:-----------------------------|:------------|
+| UI_BASE_URI     | ui.base.uri                  | http://localhost:8080/api |
+| UI_MODE         | ui.mode                      | production |
+| DEPLOYMENT_MODE | deployment.mode              | validation.disabled |
+
```

Comment:
Could you add a real explanation for the config properties and maybe move the default values into an extra column?